cg_hermes/cli/validate.py

```diff
@@ -27,11 +27,11 @@
 @app.command("deliverables")
 def validate_deliverables(
     infile: Path,
-    pipeline: Workflow = typer.Option(Workflow.FLUFFY, help="Specify pipeline"),
+    pipeline: Workflow = typer.Option(Workflow.FLUFFY, help="Specify workflow"),
```

Comment:
I assume the renaming of this variable is for a future PR

Reply:
Correctly assumed. Trying to limit the scope when possible.